src/common/turbo.ts

```diff
@@ -292,23 +294,51 @@ export class TurboAuthenticatedClient
 
   /**
    * Signs and uploads raw file data to the Turbo Upload Service.
+   *
+   *    * @example using a file or path
+   * ```ts
+   * const response = await turbo.uploadFile({
+   *   file: new File([new Uint8Array([1, 2, 3])], 'test.txt'), // or a path to a file eg "/path/to/file.txt"
```

Comment:
Nit: I think putting literal data into a file object might create confusion for the uninitiated and cause people to think that they should do this rather than use the uploadData API. I think the example would be more instructive if it had some pseudocode like:
```
const file = await pickFileFromSystem(); // an example File picking function
```

Reply:
And it might not hurt to separate the File and filepath examples, potentially calling out their intended usage platforms.